Fatras/include/ActsFatras/Physics/PhotonConversion/PhotonConversion.hpp

```diff
@@ -0,0 +1,311 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Units.hpp"
+#include "Acts/Material/MaterialSlab.hpp"
+#include "Acts/Utilities/UnitVectors.hpp"
+#include "ActsFatras/EventData/Particle.hpp"
+#include "ActsFatras/EventData/ProcessType.hpp"
+#include "ActsFatras/Utilities/ParticleData.hpp"
+
+#include <limits>
+#include <random>
+#include <vector>
+
+#include <math.h>
+
+namespace ActsFatras {
+/// This class handles the photon conversion. It evaluates the distance
+/// after which the interaction will occur and the final state due the
+/// interaction itself.
+class PhotonConversion {
+ public:
+  using Scalar = ActsFatras::Particle::Scalar;
+  /// Lower energy threshold for produced children
+  Scalar minimumMomentum = 100. * Acts::UnitConstants::MeV;
```

Comment:
I still do not think that this should be configurable. The selection cut whether to simulate a particle or not is already handled at a higher level; adding it here just introduces possible inconsistencies.

I could understand if the parametrization has a lower validity threshold and there is the physical threshold of `2 * m_e < E`. Both would be fixed values and should not be configurable.

Reply:
I picked up this parameter from Athena and of course, the selection for the simulation is managed in the `Simulator`. Here, the lower threshold you mentioned could be used anywhere. However, I assume that they introduced it in order avoid some turn-on effects for energies around `2m_e` and the rapidly changing cross section. Since I didn't verify it, this is a pure assumption and I change it to the fixed value

Reply:
Ok, thanks.